components/server/src/messaging/local-message-broker.ts

```diff
@@ -110,7 +121,13 @@ export class LocalRabbitMQBackedMessageBroker implements LocalMessageBroker {
         this.disposables.push(
             this.messageBusIntegration.listenForWorkspaceInstanceUpdates(
                 undefined,
-                (ctx: TraceContext, instance: WorkspaceInstance, userId: string | undefined) => {
+                async (ctx: TraceContext, instance: WorkspaceInstance, userId: string | undefined) => {
+                    const enabled = await this.isRedisPubSubByTypeEnabled("workspace-instance");
```

Comment:
I think we should inverse check the flag on the old instance updates as well so we don't send updates twice when this is anebaled.

Reply:
Not sure I follow, this is the old message bus update